src/www/start.action.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
How are we going to use this file? Shall we update the README as well?

Reply:
it's an `.action.sh` - so it can be run with `npm run action <path/to/action>`

I'll update the README once I know @fortuna is cool with the overall approach, here